src/coreComponents/mesh/generators/CellBlockUtilities.cpp

```diff
@@ -27,7 +27,8 @@ static localIndex getFaceNodesHex( localIndex const faceNum,
                                    arraySlice1d< localIndex const, cells::NODE_MAP_USD-1 > const & elemNodes,
                                    Span< localIndex > const faceNodes )
 {
-  GEOS_ERROR_IF_LT( faceNodes.size(), 4 );
+  GEOS_ERROR_IF_LT_MSG( faceNodes.size(), 4,
+                        "Invalid number of nodes for face (face index = " << faceNum << ")." );
```

Comment:
Please put the error msg in factor (`err = "Invalid number of nodes for face (face index = {})."`) and use `GEOS_FMT`.

Reply:
Are you suggesting to do this?
```cpp
static localIndex getFaceNodesHex( localIndex const faceNum,
                                   arraySlice1d< localIndex const, cells::NODE_MAP_USD-1 > const & elemNodes,
                                   Span< localIndex > const faceNodes )
{
  std::string_view err;
  if (faceNodes.size(), 4,)
  {
    err="Invalid number of nodes for face (face index = {}).";
  }
  switch( faceNum )
  {
    case 0:

    [...]

    default:
    {
      err="Invalid number of nodes for face (face index = {}).";
    }
  }
  GEOS_ERROR_IF( err.empty(), GEOS_FMT( err, faceNum ) );
  return 4;
}
```

Reply:
I made another proposal in the last commit: storing the messages in `string_views` and reusing them as you suggested.